src/terragraph-e2e/e2e/config/config_metadata.json

```diff
@@ -2826,6 +2826,18 @@
       "action": "RESTART_STATS_AGENT",
       "readOnly": true
     },
+    "nodeName": {
+        "desc": "The name of this node",
+        "type": "STRING",
+        "action": "RELOAD_TOPOLOGY_NAMES",
+        "readOnly": true
+    },
+    "topologyName": {
+      "desc": "The name of the topology that this node belongs to",
+      "type": "STRING",
+      "action": "RELOAD_TOPOLOGY_NAMES",
+      "readOnly": true
+     },
```

Comment:
You cannot use duplicate JSON keys, this will crash every service trying to load the file.

Reply:
Acknowledged, working on review comments.

Reply:
Addressed the comments